site/static/main.css

```diff
@@ -239,6 +239,37 @@ nav a:hover {
   background-repeat: no-repeat;
 }
 
+/* Gallery */
+
+.gallery {
```

Comment:
can you also add some code to not underline the links on hover. I think it looks nicer